modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -4,19 +4,30 @@
 
 :_content-type: PROCEDURE
 [id="virt-checking-cluster-dpdk-readiness_{context}"]
-= Checking cluster readiness to run DPDK applications with zero packet loss
+= DPDK checkup
 
-Use a predefined checkup to verify if your {product-title} cluster node can run a virtual machine (VM) with a Data Plane Development Kit (DPDK) workload. The checkup runs traffic between a traffic generator pod and a VM running a test DPDK application, and checks for packet loss.
+Use a predefined checkup to verify that your {product-title} cluster node can run a virtual machine (VM) with a Data Plane Development Kit (DPDK) workload with zero packet loss. The DPDK checkup runs traffic between a traffic generator pod and a VM running a test DPDK application.
+
+You run a DPDK checkup by performing the following steps:
+
+. Create a service account, role, and role bindings for the DPDK checkup and a service account for the traffic generator pod.
+. Create a security context constraints resource for the traffic generator pod.
+. Create a config map. The config map provides the input to run the checkup and stores the results.
```

Comment:
s/Create a config map. The config map provides/Create a config map to provide (minimalism)